lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -235,6 +235,19 @@ static SmallVector<unsigned> eraseOrder(ArrayRef<unsigned> order,
   return resOrder;
 }
 
+SmallVector<unsigned> getMatrixOrder(unsigned rank, bool rowMajor) {
```

Comment:
I'm a bit confused about the name "matrix" when rank > 2

Reply:
We consider it as a batched matrix of shape `[*, m, n]`, where * are zero or more dimensions (see the comment below).